test/integration/driver.go

```diff
@@ -432,6 +460,44 @@ func testRouting() error {
 	return nil
 }
 
+func waitForNewRestartEpoch(pod string, start int) error {
+	log.Println("Waiting for Envoy restart epoch to increment...")
+	for n := 0; n < budget; n++ {
+		current, err := getRestartEpoch(pod)
+		if err != nil {
+			log.Printf("Could not obtain Envoy restart epoch for %s: %v", pod, err)
+		}
+
+		if current > start {
+			return nil
+		}
+		time.Sleep(1 * time.Second)
+		n++
```

Comment:
'n' is incremented twice per loop: once here and once in the 'for' post statement one line 465